fvm/evm/emulator/state/base.go

```diff
@@ -356,25 +356,54 @@ func (v *BaseView) PurgeAllSlotsOfAnAccount(addr gethCommon.Address) error {
 	if err != nil {
 		return err
 	}
-	if acc == nil { // if account doesn't exist return
+	// if account doesn't exist, return
+	if acc == nil || len(acc.CollectionID) == 0 {
```

Comment:
len(acc.CollectionID) == 0 means the account is a non smart contract account, which has no slots to purge, right? better to add this as comments

Reply:
Good point :+1: 
Added in https://github.com/onflow/flow-go/pull/7737/commits/16ee161c5ba5628a9f69b4a550258aa1773ff31d